### DISCUSSION TITLE
Google Sheets to CSV Transform Feedback and Discussion

### DISCUSSION THREAD

**Discussion by CECSpecialistI** [crystalyragui/MARC2RDA#369] [General]
This is a place for comments and discussion RE: [CSV output](https://github.com/uwlib-cams/MARC2RDA/tree/main/Working%20Documents/Draft%20Field%20By%20Field%20Spreadsheets/csv) review.

**Comment by CECSpecialistI** [comment]
@gerontakos Notes from 2022-06-22: <br>
Crystal checked [700](https://github.com/uwlib-cams/MARC2RDA/blob/main/Working%20Documents/Draft%20Field%20By%20Field%20Spreadsheets/csv/007.csv#L1), noticed all notes are not present (was that on purpose?) and an extra column to the left of "Status". 

@SitaKB noticed the extra column as well, and some notes in the [008](https://github.com/uwlib-cams/MARC2RDA/blob/main/Working%20Documents/Draft%20Field%20By%20Field%20Spreadsheets/csv/008.csv#L1068), including things like "From RDA to MARC mapping: rdam:P30001 rdakit:hasM21 "MARC 21 Bibliographic 008 /29 [identifier] (maps, visual materials)" ." @mcm104 did you add these notes to the 008 when you edited recently? When I removed the filter on the Google sheet, I was able to see the notes in question in Google Sheets, so this could be a non-issue.

**Reply by SitaKB** [reply]
Is a non-issue.

**Reply by mcm104** [reply]
Yes, these notes were added when I was correcting the 00X spreadsheets!

These comments are on rows that contained mappings from the RDA-to-MARC mapping that I had to manually add into the spreadsheet. (Anything from the RDA-to-MARC mapping that was already in the spreadsheet does _not_ have this comment.) Because these rows are all new additions, I wanted to add a comment to show A) this is a new mapping, B) it has not been evaluated yet going from MARC to RDA, and C) here's what the RDA-to-MARC mapping wrote specifically, in case it seems weird/confusing.

If desired, these notes can be deleted as these rows are reviewed!

**Comment by lake44me** [comment]
@lake44me checked 500 and 521, and didn't notice anything missing or out of place.

It's really difficult to navigate and eyeball both things in a browser window on the same screen.  I did the best I could.

**Reply by CECSpecialistI** [reply]
Thanks Laura and Melissa!

**Comment by SitaKB** [comment]
It is a non-issue. Melissa added the notes when she recently edited it.

Van: Crystal Clements ***@***.***>
Verzonden: woensdag 22 juni 2022 19:33
Aan: uwlib-cams/MARC2RDA ***@***.***>
CC: Sita Bhagwandin ***@***.***>; Mention ***@***.***>
Onderwerp: Re: [uwlib-cams/MARC2RDA] Google Sheets to CSV Transform Feedback and Discussion (Discussion #369)


@gerontakos<https://eur06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fgerontakos&data=05%7C01%7Csita.bhagwandin%40kb.nl%7C4530354479cb4637dc1408da54753ffb%7C01ef09a2c6084fc7af66836e50ddde2c%7C0%7C0%7C637915159804263234%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=X5WhkKep%2BddjQlS5vdXOwDnUhturBSz1KFhh1tkt1zE%3D&reserved=0> Notes from 2022-06-22:

Crystal checked 700<https://eur06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fuwlib-cams%2FMARC2RDA%2Fblob%2Fmain%2FWorking%2520Documents%2FDraft%2520Field%2520By%2520Field%2520Spreadsheets%2Fcsv%2F007.csv%23L1&data=05%7C01%7Csita.bhagwandin%40kb.nl%7C4530354479cb4637dc1408da54753ffb%7C01ef09a2c6084fc7af66836e50ddde2c%7C0%7C0%7C637915159804263234%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=X3jeXRoSSMf3TixIm6EppDrky4EnOc0K7JJQACnZkEM%3D&reserved=0>, noticed all notes are not present (was that on purpose?) and an extra column to the left of "Status".

@SitaKB<https://eur06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FSitaKB&data=05%7C01%7Csita.bhagwandin%40kb.nl%7C4530354479cb4637dc1408da54753ffb%7C01ef09a2c6084fc7af66836e50ddde2c%7C0%7C0%7C637915159804263234%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=RLyLLs4ueSrKbfsT15wQnv%2BMunjImDizaKsq7KdSGdE%3D&reserved=0> noticed the extra column as well, and some notes in the 008<https://eur06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fuwlib-cams%2FMARC2RDA%2Fblob%2Fmain%2FWorking%2520Documents%2FDraft%2520Field%2520By%2520Field%2520Spreadsheets%2Fcsv%2F008.csv%23L1068&data=05%7C01%7Csita.bhagwandin%40kb.nl%7C4530354479cb4637dc1408da54753ffb%7C01ef09a2c6084fc7af66836e50ddde2c%7C0%7C0%7C637915159804263234%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=qFjpejFt24SImnFARLV0TD6KmvE0mj2JYx3ecGw3LVM%3D&reserved=0>, including things like "From RDA to MARC mapping: rdam:P30001 rdakit:hasM21 "MARC 21 Bibliographic 008 /29 [identifier] (maps, visual materials)" ." @mcm104<https://eur06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmcm104&data=05%7C01%7Csita.bhagwandin%40kb.nl%7C4530354479cb4637dc1408da54753ffb%7C01ef09a2c6084fc7af66836e50ddde2c%7C0%7C0%7C637915159804263234%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=sXiUi%2BZVYmSA%2FTEdkWB%2F5tYBCh5m7Vnyc11pozFTmzU%3D&reserved=0> did you add these notes to the 008 when you edited recently? When I removed the filter on the Google sheet, I was able to see the notes in question in Google Sheets, so this could be a non-issue.

-

**Reply by CECSpecialistI** [reply]
Thank you Sita!
